assets/src/edit-story/karma/fixture/containers/designPanel/textStyle.js

```diff
@@ -81,10 +81,6 @@ export class TextStyle extends AbstractPanel {
     return this.getByRole('textbox', { name: /Font size/ });
   }
 
-  get collapse() {
-    return this.getByRole('button', { name: /Collapse panel/ });
```

Comment:
Instead of removing this & its usage in `assets/src/edit-story/components/panels/karma/textStyle.karma.js`, would it be possible to replace this with

```js
get collapse() {
  return this.getByRole('button', { name: 'Style' });
}
```

?

Reply:
That's a good catch! Tested and verified!